types/src/on_chain_config/consensus_config.rs

```diff
@@ -218,6 +218,11 @@ impl OnChainConsensusConfig {
         true
     }
 
+    /// Returns the execution pool window size. None if execution pool is disabled.
+    pub fn execution_pool_window_size(&self) -> Option<u64> {
+        None // TODO: implement me!
+    }
+
```

Comment:
i think once my changes are merged we can rebase and remove

Reply:
Awesome, I'll update this.